File: LeetCode/Math/Easy/common_factors.py
class Solution:
    def commonFactors(self, a: int, b: int) -> int:
        aa = []
        bb = []
        cc = []
        for i in range(1, a+1):
            if a%i == 0:
                aa.append(i)
        for i in range(1, b+1):
            if b%i == 0:
                bb.append(i)
        
        for i in aa:
            if i in bb:
                cc.append(i)
        return len(cc)

File: LeetCode/Math/Easy/common_factors.c
int commonFactors(int a, int b){
    int i = 1;
    int j = 0;
    if (a >= b){
        while (i <= a){
            if (a%i == 0 && b%i==0) j += 1;
            i++;}}
    else{
        while (i <= b){
            if (a%i==0 && b%i==0) j+=1;
            i++;}}
    return (j);
}
